java/src/org/certificatetransparency/ctlog/SignedTreeHead.java

```diff
@@ -0,0 +1,16 @@
+package org.certificatetransparency.ctlog;
+
+import org.certificatetransparency.ctlog.proto.Ct;
+
+
+public class SignedTreeHead {
+  public Ct.Version version;
+  public long timestamp;
+  public long treeSize;
+  public byte[] sha256RootHash;
+  public Ct.DigitallySigned signature;
+
+  public SignedTreeHead(Ct.Version version_) {
```

Comment:
Since I'm not 100% sure of the behaviour of uninitialised enums in Java, I decided to make it a non-problem by ensuring it *couldn't* be uninitialised. 😃